StRoot/StEvent/StBTofHeader.h

```diff
@@ -95,6 +115,17 @@ class StBTofHeader : public StObject {
     Short_t      mNTzeroCan;
     Double_t     mTCanFirst;
     Double_t     mTCanLast;
+    Double_t     mVpdEHits;
+    Double_t     mVpdWHits;
+    Double_t     mVpdEGoodHits;
+    Double_t     mVpdWGoodHits;
+    Double_t     mEarliestVpdEHit;
+    Double_t     mEarliestVpdWHit;
+    Double_t     mClosestVpdEHit;
+    Double_t     mClosestVpdWHit;
+    Double_t     mLatestVpdEHit;
+    Double_t     mLatestVpdWHit; 
+
     ClassDef(StBTofHeader,3)
```

Comment:
```suggestion
    ClassDef(StBTofHeader,4)
```

Reply:
I don't see a strong need for this and propose to leave as is: the increment to 3 was done a few days ago and no library or production has been based on it since.